### ISSUE TITLE
Add auto-grow flag to azurerm_mysql_server and azurerm_mariadb_server resources

### ISSUE THREAD

**Issue by marcoboffi** [terraform-providers/terraform-provider-azurerm#4295]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

Add new argument to azurerm_mysql_server and azurerm_mariadv_server resources to enable auto-grow feature.

### New or Affected Resource(s)

* azurerm_mysql_server
* azurerm_mariadb_server

### Potential Terraform Configuration

```hcl
# Copy-paste your Terraform configurations here - for large Terraform configs,
# please use a service like Dropbox and share a link to the ZIP file. For
# security, you can also encrypt the files using our GPG public key.
```

### References

* #0000

**Comment by r0bnet** [comment]
MariaDB PR: https://github.com/terraform-providers/terraform-provider-azurerm/pull/4302

**Comment by manicminer** [comment]
Fixed with #4302 & #4303
